src/replit_river/server_transport.py

```diff
@@ -33,6 +32,8 @@
 from replit_river.session import Session
 from replit_river.transport_options import TransportOptions
 
+PROTOCOL_VERSION = "v1.1"
```

Comment:
maybe a comment here that only client is v2, server is still v1.1

Reply:
indicated by the package and the presence of `PROTOCOL_VERSION` in the different files where it matters, yes.